### PR TITLE
fix alignment and translate issues

### PR DESCRIPTION
**Pull request by renancaraujo**
This fix this issue: #4 

Also, change width and height to be defined on paint time.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## [x] - Next
+
+- Fix alignment issue
+- Fix positioning issues
+- Change width and height to be defined on animations paint time (breaking).
+
+
```
Comment (erickzanardo) [inline CHANGELOG.md]: One blank line is enough :)

File: example/lib/main.dart [hunk 2]
```diff
@@ -93,20 +93,31 @@ class MinionComponent extends FlareActorComponent with Resizable {
...
+            controller: minionController,
+            fit: BoxFit.scaleDown,
+            alignment: Alignment.center,
+          ),
+          width: 300,
+          height: 300,
+        );
+
+  @override
+  void render(Canvas c) {
```
Comment (erickzanardo) [inline example/lib/main.dart:99]: Suggestion: If you mark the basegame as debugmode true (check the docs on flame on how to) the bounding box will be drawn automatically

File: lib/src/flare_actor_component.dart [hunk 3]
```diff
@@ -19,8 +26,8 @@ class FlareActorComponent extends PositionComponent {
   @override
   void render(Canvas canvas) {
     prepareCanvas(canvas);
-    canvas.save();
-    flareAnimation.render(canvas, toPosition().toOffset());
+    final size = ui.Size(width, height);
```
Comment (erickzanardo) [inline lib/src/flare_actor_component.dart]: Maybe we could memoize this? Just a small suggestion, it's up to you

File: lib/src/flare_particle.dart [hunk 4]
```diff
@@ -4,20 +4,23 @@ part of flame_flare;
...
   }
 
   @override
   void render(Canvas canvas) {
     canvas.save();
-    flareAnimation.render(
-      canvas,
-      Offset(-flareAnimation.width / 2, -flareAnimation.height / 2),
-    );
+    final size = ui.Size(width, height);
```
Comment (erickzanardo) [inline lib/src/flare_particle.dart:27]: Same suggestion about the memoization

**Review by erickzanardo — APPROVED** [review]
A few suggestions, but LGTM already

File: CHANGELOG.md (hunk 1 above)
Comment (renancaraujo) [reply CHANGELOG.md]: The human linter 2, whitespace strikes back.

Comment (erickzanardo) [reply CHANGELOG.md]: Come on, extra blank lines are annoying :P

File: lib/src/flare_particle.dart (hunk 4 above)
Comment (renancaraujo) [reply lib/src/flare_particle.dart:27]: done

File: example/lib/main.dart (hunk 2 above)
Comment (renancaraujo) [reply example/lib/main.dart:99]: Yep, it is actvated

File: example/pubspec.lock [hunk 5]
```diff
@@ -171,7 +178,7 @@ packages:
       name: petitparser
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "3.1.0"
+    version: "3.0.4"
```
Comment (spydon) [inline example/pubspec.lock:181]: Why all the dependency downgrades?

**Review by spydon — APPROVED** [review]

**Pull request merged**